openslides/assignments/config_variables.py

```diff
@@ -33,7 +33,7 @@ def get_config_variables():
 
     yield ConfigVariable(
         name="assignments_poll_100_percent_base",
-        default_value="YES_NO_ABSTAIN",
+        default_value="VALID",
```

Comment:
This assumes that one adds values to the valid field. But in ad hoc elections this is not always true. So I prefer YES_NO_ABSTAIN.

Reply:
Ok, changed.